deepdrivemd/sim/openmm_stream/run_openmm.py

```diff
@@ -0,0 +1,207 @@
+import shutil
+import simtk.unit as u
+import simtk.openmm as omm
+from mdtools.openmm.sim import configure_simulation
+from deepdrivemd.utils import Timer, parse_args
+from deepdrivemd.sim.openmm_stream.config import OpenMMConfig
+from deepdrivemd.sim.openmm_stream.openmm_reporter import ContactMapReporter
+import sys
+import os
+import time
+import random
+from lockfile import LockFile
+import pickle
+import parmed as pmd
+import numpy as np
+import subprocess
+import itertools
+from deepdrivemd.sim.openmm.run_openmm import SimulationContext
+
+
+def configure_reporters(
+    sim: omm.app.Simulation,
+    ctx: SimulationContext,
+    cfg: OpenMMConfig,
+    report_steps: int,
+):
+
+    sim.reporters.append(ContactMapReporter(report_steps, cfg))
+
+
+def next_outlier(cfg: OpenMMConfig, sim: omm.app.Simulation):
+    """Get the next outlier to use as an initial state.
+
+    Parameters
+    ----------
+    cfg : OpenMMConfig
+    sim : omm.app.Simulation
+
+    Returns
+    -------
+    Tuple[str, str, float, str]
+        path to pdb file with positions, path to numpy file with velocities, rmsd, md5sum
+
+    """
+    if cfg.next_outlier_policy == 1:
+        cfg.pickle_db = cfg.outliers_dir + "/OutlierDB.pickle"
+
+        if not os.path.exists(cfg.pickle_db):
+            return None
+
+        if cfg.lock == "set_by_deepdrivemd":
+            cfg.lock = LockFile(cfg.pickle_db)
+
+        cfg.lock.acquire()
+        with open(cfg.pickle_db, "rb") as f:
+            db = pickle.load(f)
+        md5 = db.sorted_index[cfg.task_idx]
+        rmsd = db.dictionary[md5]
+        positions_pdb = cfg.outliers_dir + f"/{md5}.pdb"
+        velocities_npy = cfg.outliers_dir + f"/{md5}.npy"
+        shutil.copy(positions_pdb, cfg.current_dir)
+        shutil.copy(velocities_npy, cfg.current_dir)
+        shutil.copy(cfg.pickle_db, cfg.current_dir)
+        cfg.lock.release()
+
+        with open(cfg.current_dir + "/rmsd.txt", "w") as f:
+            f.write(f"{rmsd}\n")
+
+        positions_pdb = cfg.current_dir + f"/{md5}.pdb"
+        velocities_npy = cfg.current_dir + f"/{md5}.npy"
+
+        return positions_pdb, velocities_npy, rmsd, md5
+
+    elif cfg.next_outlier_policy == 0:
+        # TODO
+        return None
```

Comment:
This should raise a ValueError right? Otherwise the program will never set a new restart point.

Reply:
For now only one policy is used, so if, elif is removed.